services/api/src/admin_intersection.py

```diff
@@ -65,30 +84,65 @@ def get_intersection_data(intersection_id):
         return intersection_list
 
 
-def get_modify_intersection_data(intersection_id):
+def get_modify_intersection_data(
```

Comment:
question: do we have a separate task for updating documentation and method descriptions or should we do that while we're in here?

Reply:
I am trying not to go too crazy since we will be re-writing most of this in Java at some point, but I did add some additional docstring comments. The auth_tools method is well document, and I have attempted to use variable names and typing to self-document the arguments